packages/integration_test/test/binding_fail_test.dart

```diff
@@ -61,13 +61,18 @@ Future<Map<String, dynamic>> _runTest(String scriptPath) async {
   final String testResults = (await process.stdout
           .transform(utf8.decoder)
           .expand((String text) => text.split('\n'))
-          .map((String line) {
+          .map<dynamic>((String line) {
```

Comment:
Not sure you need this `<dynamic>` here? But if no linter complains, let's gooo!

Reply:
Yeah I think since this is copied from the SDK repo, where type annotations are necessary